src/control/client/connect.go

```diff
@@ -74,6 +74,7 @@ type Connect interface {
 	KillRank(uuid string, rank uint32) ResultMap
 	PoolCreate(*PoolCreateReq) (*PoolCreateResp, error)
 	PoolDestroy(*PoolDestroyReq) error
+	PoolGetACL(uuid string) ([]string, error)
```

Comment:
this area will probably change fairly soon but we may want to have req/resp param structs for each to be consistent

Reply:
I was wondering about this, but noticed that some other calls in this API don't have req/resp structs when there would only be a single member in each. If we're moving in that direction, I'll happily add the structs.